lib/python/qmk/cli/__init__.py

```diff
@@ -4,6 +4,7 @@
 """
 from . import cformat
 from . import compile
+from . import convert
```

Comment:
This seems ambiguous as the CLI command name. Perhaps it should be kle2qmk as well?

Reply:
I agree! I will change that now